alma/lib/Services/InsuranceProductService.php

```diff
@@ -217,9 +236,7 @@ public function addInsuranceProduct(
             $this->context->shop->id
         );
 
-        if ($destroyPost) {
-            $_POST['alma_id_insurance_contract'] = null;
-        }
+        $_POST['alma_id_insurance_contract'] = null;
```

Comment:
??

Reply:
I think it was for PS 1.6. I remove it and we will tested it :)